pyblish/plugin.py

```diff
@@ -462,10 +503,38 @@ def process(plugin, context, instance=None, action=None):
 
     """
 
-    if issubclass(plugin, (ContextPlugin, InstancePlugin)):
-        result = __explicit_process(plugin, context, instance, action)
-    else:
-        result = __implicit_process(plugin, context, instance, action)
+    if not context.get('profiling'):
```

Comment:
Typo here, should be `context.data.get()`, as below.

Unsure about whether `profiling` is the best name here. Grammatically it's a verb, describing some sort of state. That it is currently "profiling". As it isn't meant as that, it should probably we `profile`, but even that could be confused with an action. To "profile" something.

I'm thinking `context.data["statistics"]`

Reply:
I agree, *statistics* is much better.